wordpress-mobile/gutenberg-mobile
# DarkMode improvements

This PR removes the need to pass this.props.theme to useStyle on every instance where withTheme is used.

`gutenberg` PR: https://github.com/WordPress/gutenberg/pull/17309

To test:
- Run the iOS example app using Xcode 11 (beta)
- Set the simulator/device to DarkMode (might need to build/run the example app again)
- Check that all colors looks good and there are no errors.
  - All blocks
  - Bottom-sheets (Add link - Image settings)
  - Block inserter
  - Unsupported/Validation error placeholders
  - HTML mode
